Snakefile

```diff
@@ -434,7 +448,7 @@ rule ploidetect_copynumber:
     conda:
         "conda_configs/r.yaml"
     container:
-        "docker://lculibrk/ploidetect"
+        "docker://lculibrk/ploidetect:devel"
```

Comment:
Does this actually still need the different docker container? (the :devel?)

Reply:
Didn't mean to push that, whoops

Reply:
But it reminds me that I need to rebuild & retest the base image after some tangles I got myself into on the Ploidetect repo

Reply:
Reverted in 12bee71